src/Login.ts

```diff
@@ -235,9 +235,12 @@ export class Login {
   public static async validateJWT(token: string): Promise<TokenStatus> {
     const url = `${loginUrl()}/token`;
     const params = { token };
+    const authHeader = instance.defaults.headers.Authorization;
```

Comment:
We can use `setBearerToken('')` here instead

Reply:
Just set `setBearerToken(accessToken);` only after `Login.validateJWT(idToken)`

Reply:
It will not work since if you loggin the second time the Authorization header is already set. The request will fail if `Authorization: Bearer `. It needs to be completely removed.

Reply:
Alright, hope you know what you are doing :)